src/redux/actions/books.js

```diff
@@ -0,0 +1,45 @@
+// constants
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const TOGGLE_BOOK = 'TOGGLE_BOOK';
+
+// action creators
+function addBook(book) {
```

Comment:
Please ensure that the structure of your files should follow the `ducks pattern` like the code below according to the project requirement:

```
   ...
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js
```